### PR TITLE
WIP update linting and add a11y linter with fixes

### PR DESCRIPTION
**Pull request by phillimorland**
- [x] remove strupstrop
- [x] remove semi colon
- [x] look into rule for if/else stick to if/else outside of render
- [ ] update docs
- [x] add line length 80

### CODE REVIEW

File: components/MenuItem.js [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
+import { TouchableWithoutFeedback } from 'react-native'
+import { ListItem } from '@rneui/themed'
+import IconSLI from 'react-native-vector-icons/SimpleLineIcons'
+import { COLORS } from '../theme/colors'
+import { SIZES } from '../theme/fonts'
+import { styles } from './MenuItem.styles'
 
 export const MenuItem = (props) => {
-  const {item: { isCancel, isFirst, isLast, isAllowed }} = props;
+  const {item: { isCancel, isFirst, isLast, isAllowed }} = props
```
[the remark this answers is not in the document]
Comment (khamui) [reply components/MenuItem.js:10]: just read the question now. you mean the `destructuring` of objects (and arrays?) right?
I am not sure, sometimes i find it informative to know which variables are coming from props.

but sometimes it is just too long to always add the props. in front. Not sure, personally I am fine with both. What do the others think? @goapunk @Rineee @fuzzylogic2000 ?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

File: .eslintrc.json [hunk 2]
```diff
@@ -31,11 +32,18 @@
...
       "quotes": ["error", "single"],
-      "semi": ["error", "always"],
-      "no-unused-vars": ["warn", {"varsIgnorePattern": "(React)"}],
+      "semi": ["error", "never"],
+      "no-unused-vars": ["error", {"varsIgnorePattern": "(React)"}],
       "eol-last": ["error", "always"],
       "no-multiple-empty-lines": ["error", {"max": 1}],
-      "brace-style": ["error", "stroustrup", { "allowSingleLine": true}],
+      "brace-style": ["error", "1tbs", { "allowSingleLine": true}],
+      "max-len": ["warn", 100, 2, {
```
[the remark this answers is not in the document]
Comment (goapunk) [reply .eslintrc.json:40]: 100 is fine for me

File: components/MenuItem.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (goapunk) [reply components/MenuItem.js:10]: same here, I think both has its usage so I'd vote for allow both.

File: .eslintrc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (khamui) [reply .eslintrc.json:40]: okay for me too

File: .eslintrc.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
       "no-unused-vars": ["error", {"varsIgnorePattern": "(React)"}],
       "eol-last": ["error", "always"],
       "no-multiple-empty-lines": ["error", {"max": 1}],
-      "brace-style": ["error", "stroustrup", { "allowSingleLine": true}],
+      "brace-style": ["error", "1tbs", { "allowSingleLine": true}],
```
Comment (khamui) [inline .eslintrc.json:39]: auf wiedersehen stroustrup 🙃

Comment (phillimorland) [reply .eslintrc.json:39]: there can be only one! :p

File: components/MenuItem.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (phillimorland) [reply components/MenuItem.js:10]: err feels messy but cool

**Pull request closed without merging**